src/views/partials/header.ejs

```diff
@@ -26,6 +25,12 @@
     <link rel="stylesheet" href="/css/accessible-autocomplete.min.css">
     <link rel="stylesheet" href="/css/app.css">
     <meta property="og:image" content="/images/govuk-opengraph-image.png">
+    <meta name="theme-color" content="#323232">
+    <meta name="theme-color" content="#323232" media="(prefers-color-scheme: light)">
+    <meta name="theme-color" content="#323232" media="(prefers-color-scheme: dark)">
+    <meta name="msapplication-navbutton-color" content="#323232">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="#323232">
```

Comment:
Is this theme styling requested by UX?

Reply:
It fixes a bug in the styling... It also only affects Safari.  It's not something cover in the GEL at all which feels a bit odd.  However it does stop safaris toolbar turning bright blue when you visit the site.